.eslintrc

```diff
@@ -4,42 +4,58 @@
     "no-undef": 2,
     "no-var": 1,
     "no-unused-vars": [
-      1,
-      { "vars": "all", "args": "none", "varsIgnorePattern": "^_.+" }
+      "error",
+      {
+        "vars": "all",
+        "args": "none",
+        "varsIgnorePattern": "^_.+",
+        "ignoreRestSiblings": true
```

Comment:
Somehow `@typescript-eslint/no-unused-vars` will complain if I didn't add this rule there, I'm not sure why the JS rule doesn't need this. But I add them both to stay consistent as the typescript version seems to reflect all options that the main `no-unused-vars` has anyway.

However, it's worth noting that `destructuredArrayIgnorePattern` only exists in the later ESlint versions in some V8+ (we're using v7 at the moment). But upgrading ESLint to V8 leads to ESLint not runnable, and I don't want to debug that now, so I left out that option in this case.